parmed/gromacs/gromacstop.py

```diff
@@ -10,6 +10,10 @@
 from datetime import datetime
 import math
 import os
+try:
+    import pwd
+except ImportError:
+    pwd = None
```

Comment:
But why?  That doesn't make any sense...  `pwd` is imported below and `ImportError` is caught.

Should the stuff that comes after `pwd` is imported be relegated to the `else` block of the try-except?